superset/reports/notifications/email.py

```diff
@@ -177,8 +177,7 @@ def _get_content(self) -> EmailContent:
 
     def _get_subject(self) -> str:
         return __(
-            "%(prefix)s %(title)s",
-            prefix=app.config["EMAIL_REPORTS_SUBJECT_PREFIX"],
```

Comment:
Feels like this could still be useful even with per-schedule customisation, as you might want to prefix all your emails with a standard prefix and then choose individual subject lines after the prefix. I'd continue supporting it, though we could probably improve the behaviour where prefix is `None` or empty by removing any leading space etc.

Reply:
I think if the user wants to have prefix then they can always add it themselves and we should allow user 100% control over their email subject.

For the default subject line it still have prefix I just modify code to this

```
if self._report_schedule.chart:
                name = (
                    f"{prefix} "
                    f"{self._report_schedule.name}: "
                    f"{self._report_schedule.chart.slice_name}"
                )
            else:
                name = (
                    f"{prefix} "
                    f"{self._report_schedule.name}: "
                    f"{self._report_schedule.dashboard.dashboard_title}"
                )
```

Reply:
To keep backward compatibility, can we continue to include the prefix if `app.config["EMAIL_REPORTS_SUBJECT_PREFIX"]` exists or is not blank/empty/nullish? This way each application can decide for themselves if they would like to include a prefix or not.

Reply:
I agree with @giftig on the premise that the app.config is set by the application admin, whereas the title will be set by individual users. If an application admin set an app.config prefix, it's likely something important that they don't want removed from the emails, such as "Company Confidential", etc.

Reply:
Ok, I got it. I already modify to be as previous.